### PR TITLE
Enable tests in CI

### PR DESCRIPTION
**Pull request by mrskug**

### CODE REVIEW

File: apps/hbl365/package.json [hunk 1]
```diff
@@ -14,6 +14,6 @@
     "start": "parcel index.html -p 8000",
     "build": "spago build && parcel build index.html --public-url ./ --no-source-maps",
     "wait-for-parcel": "perl -e '$a = 1; while ($a) { $a = system(\"curl http://localhost:8000/\"); sleep(1)}'",
-    "test": "spago build && yarn install --cache-folder ../../.yarn-cache && yarn start & yarn wait-for-parcel && spago test"
+    "test": "#spago build && yarn install --cache-folder ../../.yarn-cache && yarn start & yarn wait-for-parcel && spago test"
```
Comment (vapaj) [inline apps/hbl365/package.json:17]: Intentional `"#"`?

File: apps/mosaico/test.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```
[the remark this answers is not in the document]
Comment (vapaj) [reply apps/mosaico/test.sh:1]: Answer: yes.

File: apps/hbl365/package.json (hunk 1 above)
Comment (vapaj) [reply apps/hbl365/package.json:17]: Answer: yes.

**Review by vapaj — APPROVED** [review]

File: apps/hbl365/package.json (hunk 1 above)
Comment (mrskug) [reply apps/hbl365/package.json:17]: yes

File: apps/mosaico/test.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (mrskug) [reply apps/mosaico/test.sh:1]: yes

**Pull request merged**